plugins/node/instrumentation-amqplib/src/types.ts

```diff
@@ -15,14 +15,14 @@
  */
 import { Span } from '@opentelemetry/api';
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
-import type * as amqp from 'amqplib';
+import { ConsumeMessage, Options } from './internal-types';
```

Comment:
It's not permitted to import `internal-types` in `types`. Everything public must go into the `types.ts` file. This practice helps avoid leaking anything internal (that could depend on dev dependencies) into the public api.

Reply:
You can read more about it [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/GUIDELINES.md#exporting-1)